Crash on receiving MMS

```
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:304)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.AssertionError: java.util.concurrent.ExecutionException: java.io.FileNotFoundException: PartAuthority couldn't load Uri resource.
	at org.smssecure.smssecure.notifications.SingleRecipientNotificationBuilder.getBigPicture(SingleRecipientNotificationBuilder.java:200)
	at org.smssecure.smssecure.notifications.SingleRecipientNotificationBuilder.build(SingleRecipientNotificationBuilder.java:147)
	at org.smssecure.smssecure.notifications.MessageNotifier.sendSingleThreadNotification(MessageNotifier.java:211)
	at org.smssecure.smssecure.notifications.MessageNotifier.updateNotification(MessageNotifier.java:161)
	at org.smssecure.smssecure.notifications.MessageNotifier.updateNotification(MessageNotifier.java:117)
	at org.smssecure.smssecure.service.KeyCachingService$1.doInBackground(KeyCachingService.java:113)
	at org.smssecure.smssecure.service.KeyCachingService$1.doInBackground(KeyCachingService.java:106)
	at android.os.AsyncTask$2.call(AsyncTask.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more
Caused by: java.util.concurrent.ExecutionException: java.io.FileNotFoundException: PartAuthority couldn't load Uri resource.
	at com.bumptech.glide.request.RequestFutureTarget.doGet(RequestFutureTarget.java:189)
	at com.bumptech.glide.request.RequestFutureTarget.get(RequestFutureTarget.java:100)
	at org.smssecure.smssecure.notifications.SingleRecipientNotificationBuilder.getBigPicture(SingleRecipientNotificationBuilder.java:194)
	... 12 more
Caused by: java.io.FileNotFoundException: PartAuthority couldn't load Uri resource.
	at org.smssecure.smssecure.mms.DecryptableStreamLocalUriFetcher.loadResource(DecryptableStreamLocalUriFetcher.java:33)
	at org.smssecure.smssecure.mms.DecryptableStreamLocalUriFetcher.loadResource(DecryptableStreamLocalUriFetcher.java:16)
	at com.bumptech.glide.load.data.LocalUriFetcher.loadData(LocalUriFetcher.java:44)
	at com.bumptech.glide.load.model.ImageVideoModelLoader$ImageVideoFetcher.loadData(ImageVideoModelLoader.java:70)
	at com.bumptech.glide.load.model.ImageVideoModelLoader$ImageVideoFetcher.loadData(ImageVideoModelLoader.java:53)
	at com.bumptech.glide.load.engine.DecodeJob.decodeSource(DecodeJob.java:170)
	at com.bumptech.glide.load.engine.DecodeJob.decodeFromSource(DecodeJob.java:128)
	at com.bumptech.glide.load.engine.EngineRunnable.decodeFromSource(EngineRunnable.java:122)
	at com.bumptech.glide.load.engine.EngineRunnable.decode(EngineRunnable.java:101)
	at com.bumptech.glide.load.engine.EngineRunnable.run(EngineRunnable.java:58)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
	at com.bumptech.glide.load.engine.executor.FifoPriorityThreadPoolExecutor$DefaultThreadFactory$1.run(FifoPriorityThreadPoolExecutor.java:118)
```

--- a/src/org/smssecure/smssecure/notifications/SingleRecipientNotificationBuilder.java
+++ b/src/org/smssecure/smssecure/notifications/SingleRecipientNotificationBuilder.java
@@ -173,8 +173,12 @@ public class SingleRecipientNotificationBuilder extends AbstractNotificationBuil
       }
 
       Slide thumbnailSlide = slideDeck.get().getThumbnailSlide();
+      Uri   uri            = thumbnailSlide.getThumbnailUri();
 
-      return thumbnailSlide != null         &&
+      DecryptableStreamUriLoader.DecryptableUri decryptableUri = new DecryptableStreamUriLoader.DecryptableUri(masterSecret, uri);
+
+      return decryptableUri != null         &&
+             thumbnailSlide != null         &&
              thumbnailSlide.hasImage()      &&
              !thumbnailSlide.isInProgress() &&
              thumbnailSlide.getThumbnailUri() != null;

